brainscore/model_interface.py

```diff
@@ -117,6 +117,31 @@ class Task:
                   - choice        (choice) object 'dog' 'cat' 'chair' 'flower' 'plane'
         """
 
+        odd_one_out = 'odd_one_out'
+        """
+        Predict the odd-one-out elements for a list of triplets of stimuli. Calling `start_task(...)` sets the 
+        similarity measure to use for the odd-one-out task. Calling `look_at(...)` with a list of triplets of stimuli
+        will compute the similarity matrix for all pairs of stimuli and compute the odd one out for each triplet.  
+        
+        Output a :class:`~brainio.assemblies.BehavioralAssembly` with the stimulus_ids as the value.
+        
+        Example:
+
+        Setting up a odd-one-out task for a list of triplets with 
+        `start_task(BrainModel.Task.odd_one_out, [image_1, image_2, image_3], similarity_measure)`
```

Comment:
This is either `'dot'` or `'cosine'` and specifies how the similarity of the activations is computed. I could probably put this in `look_at`.

Reply:
This should be decided by the model internally, not specified by the benchmark via the model interface. This should be a choice the model locks in upon construction, i.e. this could be an argument in model-tools (with a default)

Reply:
The model interface is the communication point between benchmarks and models. The idea is that any method specified in the interface could in principle be implemented on a primate (e.g. humans). Telling humans to start an odd-one-out task with `similarity_measure='dot'` does not make sense. 

Instead the similarity measure is a guess of the model for how humans perform the odd-one-out task. One such hypothesis is that cortex computes a `dot` similarity of activations. So this should be in the `ModelCommitment` in `model-tools`, not in the interface.